### PR TITLE
BTFS-2328-&-BTFS-2356

### PR DESCRIPTION
**Pull request by laipogo**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior?** (You can also refer to a JIRA ticket here)

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **What dependencies / modules need to be updated as pre-requisites?** (Include relevant PR links here)

* **Description of changes**

---

* **Please check if the PR fulfills these requirements**
- [ ] The subject of this PR contains a JIRA ticket BTFS-xxxx (N/A for community PRs)
- [ ] PR description and commit messages are [good and meaningful](https://chris.beams.io/posts/git-commit/)
- [ ] Unit tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **Please make sure the following procedures have been applied before requesting reviewers**
- [ ] Code changes closely follow [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
- [ ] Code has been *rebased* against latest master (`git merge` not recommended, unless you know what you are doing)
- [ ] Code changes have run through `go fmt`
- [ ] Code changes have run through `go mod tidy`
- [ ] All unit tests passed locally (`make test_go_test`)

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -53,6 +53,8 @@ github.com/TRON-US/go-btfs-cmds v0.2.5/go.mod h1:b1puslIW6bD50zr1gWasJSbnC7ZIJ2A
 github.com/TRON-US/go-btfs-config v0.6.0/go.mod h1:82nKCMRhsgY0I8DCasIUpSr6ZP9iHLsZJSMUxytMpEw=
 github.com/TRON-US/go-btfs-config v0.6.10 h1:I96h8iicrmUmMp1tjwt88F1WhmVfC6IS8mVb2ofTCOU=
 github.com/TRON-US/go-btfs-config v0.6.10/go.mod h1:82nKCMRhsgY0I8DCasIUpSr6ZP9iHLsZJSMUxytMpEw=
+github.com/TRON-US/go-btfs-config v0.6.11-pre4 h1:MvSTnHPlnn5V74FdfVPovoYOVzSsgrQIsmQXwpQluuw=
+github.com/TRON-US/go-btfs-config v0.6.11-pre4/go.mod h1:82nKCMRhsgY0I8DCasIUpSr6ZP9iHLsZJSMUxytMpEw=
```
Comment (taiyangc) [inline go.sum:57]: tidy?

**Comment by taiyangc** [comment]
Fix test

**Comment by taiyangc** [comment]
Build still failing.. @laipogo

**Comment by laipogo** [comment]
> Build still failing.. @laipogo

fixed....

**Pull request merged**